### ISSUE TITLE
firenvim has weird problem on firefox

### ISSUE THREAD

**Issue by gaofengnan** [glacambre/firenvim#1538]
- OS Version: MacOS Ventura 13.3.1(a)
- Browser Version: Firefox 113.0.1
- Browser Addon Version: 0.2.14 firefox add-on store version of Nov 5, 2022
- Neovim Plugin Version: newest (how do I check the version number?)

### What I tried to do
I tried to use it in Firefox. 

### What happened
I spent the past two hours trying to figure out why Firenvim does not work in Firefox, while knowing that it works very in my Edge.  What I discovered was that if I comment out the following lines in my init.vim, it works in Firefox. 
```
  let g:firenvim_config = {
      \ 'globalSettings': {
          \ 'alt': 'all',
      \  },
      \ 'localSettings': {
          \ '.*': {
              \ 'cmdline': 'neovim',
              \ 'priority': 0,
              \ 'selector': 'textarea',
              \ 'takeover': 'never',
          \ },
      \ }
  \ }
```
Please note that it works in Edge regardless whether these above lines are commented out. But why? It makes no sense.  Especially since the same above setting works well on the Firefox in my Windows machine.  I don't have /tmp/stdin or /tmp/stdout on my machine, in case you would ask me for them.

**Comment by glacambre** [comment]
I think the problem might that you have windows-style line endings in your init.vim while you need them to be unix-style on mac/linux. Could you open your init.vim in neovim, run `:set ff=unix`, `:write` the file, close it and then restart your browser and attempt to use firenvim again?

**Comment by gaofengnan** [comment]
> I think the problem might that you have windows-style line endings in your init.vim while you need them to be unix-style on mac/linux. Could you open your init.vim in neovim, run `:set ff=unix`, `:write` the file, close it and then restart your browser and attempt to use firenvim again?

No, I just tried.  I have attached both my vimrc and init.vim files. 
[vimrc.txt](https://github.com/glacambre/firenvim/files/11507040/vimrc.txt)
[init.vim.txt](https://github.com/glacambre/firenvim/files/11507039/init.vim.txt)
